docs/attributes-registry/gcp.md

```diff
@@ -6,9 +6,28 @@
 
 # GCP
 
+- [Gcp](#gcp-attributes)
+- [Gcp Client](#gcp-client-attributes)
 - [Gcp Cloud Run](#gcp-cloud-run-attributes)
 - [Gcp Gce](#gcp-gce-attributes)
 
+## GCP Attributes
+
+Attributes for Google Cloud
+
+| Attribute | Type | Description | Examples | Stability |
+| --------- | ---- | ----------- | -------- | --------- |
+
+## GCP Client Attributes
+
+Attributes for Google Cloud client libraries.
+
+| Attribute            | Type   | Description                                                                                | Examples                                              | Stability                                                        |
+| -------------------- | ------ | ------------------------------------------------------------------------------------------ | ----------------------------------------------------- | ---------------------------------------------------------------- |
+| `gcp.client.service` | string | Identifies the Google Cloud service for which the official client library is intended. [1] | `appengine`; `run`; `firestore`; `alloydb`; `spanner` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** Intended to be a stable identifier for Google Cloud client libraries that is uniform across implementation languages. The value should be derived from the canonical service domain for the service; for example, 'foo.googleapis.com' should result in a value of 'foo'. This attribute should be accompanied by 'instrumentation.source.repo' and/or 'instrumentation.source.target' when the combination of 'gcp.client.service' and 'telemetry.sdk.language' is insufficient to uniquely identify the particular client library.
```

Comment:
Since `instrumentation.source.*` attributes are not defined in the PR, maybe this sentence can be removed?

```suggestion
**[1]:** Intended to be a stable identifier for Google Cloud client libraries that is uniform across implementation languages. The value should be derived from the canonical service domain for the service; for example, 'foo.googleapis.com' should result in a value of 'foo'. 
```

Reply:
Done.